### PR TITLE
Move pins to the membership payload, we can now pin to any stream

### PR DESCRIPTION
**Pull request by texuf**

### CODE REVIEW

File: protocol/protocol.proto [hunk 1]
```diff
@@ -190,6 +191,15 @@ message MemberPayload {
         bytes token_id = 3;
     }
 
+    message Pin {
```
Comment (clemire) [inline protocol/protocol.proto:194]: backwards compatibility: If a new client (post-this-PR) sends this payload to an old stream node on omega, is that going to work?

Comment (texuf) [reply protocol/protocol.proto:194]: The request will fail until the node is upgraded.

File: protocol/protocol.proto [hunk 2]
```diff
@@ -190,6 +191,15 @@ message MemberPayload {
         bytes token_id = 3;
     }
 
+    message Pin { 
+        bytes event_id = 1;
```
Comment (ttillberg) [inline protocol/protocol.proto:195]: could you please add the userId of the creator of the pin here too?

Comment (texuf) [reply protocol/protocol.proto:195]: the creator address is in every single stream event already `bytes creator_address = 1;` it's verified against the signature of the event. Additionally, if we put it here users would be able to forge pins from other people.

Comment (texuf) [reply protocol/protocol.proto:195]: I see how we lost it, i'll fix the streamStateView_Members, one sec.

**Pull request merged**